topics/scicloud.qmd

```diff
@@ -0,0 +1,36 @@
+---
+title: SciCloud
+---
+IT for Research (ITvO) offers SciCloud, a service where server capacity can be purchased to support research applications. SciCloud virtual servers are easy to implement, easy to scale up and down and have competitive pricing based on agreed purchase. The virtual server can be placed in an external network so services running on it can be made internet accessible. It can also run on an internal network for easy security and access to some VU services.
```

Comment:
Just a though, maybe somewhere in the very beginning a short sentence explaining what it is "for dummies"? Like an example of a situation when you might need it?

Reply:
I've added "What can it be used for?" and added some typical use cases.